MAUI/Calendar/getting-started.md

```diff
@@ -84,7 +84,7 @@ namespace GettingStarted
 
 ## Change different calendar views
 
-The .NET MAUI Calendar control provides four different types of views to display dates, and it can be assigned to the control using the `View` property. By default, the control is assigned to the Month view. Initially, all Calendar views will show the current date.
+The .NET MAUI Calendar control provides four different types of views to display dates and it can be assigned to the control by using the `View` property. The control is assigned to the Month view by default. Initially, all Calendar views will show the current date.
```

Comment:
The .NET MAUI Calendar control provides four different types of views to display dates, and it can be assigned to the control using the `View` property. The control is assigned to the Month view by default. Initially, all Calendar views will show the current date.

Reply:
Content changed for calendar views